elements/lisk-p2p/src/peer_pool.ts

```diff
@@ -413,23 +413,30 @@ export class PeerPool extends EventEmitter {
 			disconnectedFixedPeers.length -
 			outboundCount;
 
+		const bestPeersbyHeight = getBestPeersbyHeight([
```

Comment:
prefer `getUniquePeersByIP` as we are going to remove the usage of height in future